internal/api/server.go

```diff
@@ -65,9 +69,18 @@ func validateConfigFile(filePath string) (Config, error) {
 	config.Key = key
 	config.DBPath = c.DBPath
 	config.Port = c.Port
+	config.PebbleNotificationsEnabled = c.Pebblenotificationsenabled
 	return config, nil
 }
 
+func SendPebbleNotification(key, request_id string) error {
+		cmd := exec.Command("pebble", "notify", key, fmt.Sprintf("request_id=%s", request_id))
+		if err := cmd.Run(); err != nil {
+			return errors.Join(errors.New("couldn't execute a pebble notify: "), err)
+		}
+		return nil
+}
+
```

Comment:
Why so much indentation?

If you use vscode you can use auto format on save to avoid this

Reply:
i think i forgot to enable it when I moved computers. fixed.